From 9f8aa48a38b339dab2cad580b5bd1a931d6ea1e5 Mon Sep 17 00:00:00 2001
From: "Vojtech Vitek (V-Teq)" <vvitek@redhat.com>
Date: Tue, 14 Feb 2012 17:57:31 +0100
Subject: [PATCH] Fix gcc signed overflow warning

---
 builtin.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/builtin.c b/builtin.c
index 4b06413..e39a7e8 100644
--- a/builtin.c
+++ b/builtin.c
@@ -1734,8 +1734,7 @@ do_mktime(NODE *tree)
 	free_temp(t1);
 
 	if (count < 6
-	    || month < month - 1
-	    || year < year - 1900 || year - 1900 != (int) (year - 1900))
+	    || year - 1900 != (int) (year - 1900))
 		return tmp_number((AWKNUM) -1);
 
 	memset(& then, '\0', sizeof(then));
-- 
1.7.6.2

