From 1f2f511f4551cd15283d9bb8eadd9f4139bd3564 Mon Sep 17 00:00:00 2001
From: "Vojtech Vitek (V-Teq)" <vvitek@redhat.com>
Date: Mon, 13 Feb 2012 22:10:49 +0100
Subject: [PATCH] Prevent overflows by increasing YYMAXDEPTH and YYSTACKSIZE
 byacc/bison parameters

---
 awkgram.y |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/awkgram.y b/awkgram.y
index 34248c7..86e96ab 100644
--- a/awkgram.y
+++ b/awkgram.y
@@ -28,6 +28,8 @@
 #define YYDEBUG 12
 #endif
 
+#define YYMAXDEPTH 10000 /* default is 500 in byacc-1.9.20070509-6.1.el6 */
+
 #include "awk.h"
 
 #if defined(__STDC__) && __STDC__ < 1	/* VMS weirdness, maybe elsewhere */
-- 
1.7.6.2

